### ISSUE TITLE
Blacklist updates do not propagate to users who already have config.json

### ISSUE THREAD

**Issue by Taudris** [Marco4413/CP77-BetterSleeves#2]
I experienced an issue where my `config.json` had no blacklisted items. I think it's because I used the mod before the default blacklist items were added, and there's no logic to add the blacklisted items to older `config.json` files.

**Comment by Taudris** [comment]
I think the least intrusive solution would be to have a JSON property that contains the full set of default blacklisted items that was known by the last version to write the config. If that property is missing a default blacklisted item, then add that item to the user's blacklist.

So for example, if a user has this `config.json` (note that `mantis_blade` is missing):

```
{
  "rollDownWeaponBlacklist": {
    "projectile_launcher": true
  },
  "rollDownWeaponDefaultBlacklist": {
    "projectile_launcher": true
  },
  "rollDownDelay": 1,
  "rollDownItemBlacklist": {
    "outfit_01__q305__hazmat_": true
  },
  "rollDownItemDefaultBlacklist": {
    "outfit_01__q305__hazmat_": true
  },
  "autoRoll": true
}
```

And `init.lua` has this:

```
function BetterSleeves:ResetConfig()
    self.autoRoll = true
    self.rollDownDelay = 1.0
    self.rollDownItemDefaultBlacklist = {
        ["outfit_01__q305__hazmat_"] = true,
    }
    self.rollDownItemBlacklist = self.rollDownItemDefaultBlacklist
    self.rollDownWeaponDefaultBlacklist = {
        ["mantis_blade"] = true,
        ["projectile_launcher"] = true,
    }
    self.rollDownWeaponBlacklist = self.rollDownWeaponDefaultBlacklist
end
```

You can identify that the user's `rollDownWeaponDefaultBlacklist` does not have `mantis_blade` in it, and therefore it needs to be added to their `rollDownItemBlacklist`.

**Comment by Marco4413** [comment]
Sorry I didn't respond earlier. I have already thought of a possible solution, which is adding a version to the config file. That will allow me to make even bigger changes to the config file and still being able to migrate from older versions. Thanks for the tip though

**Comment by Marco4413** [comment]
Should be fixed with 713ca30
